Dfe.PrepareConversions/Dfe.PrepareConversions.Tests/Mappings/CreateSponsoredProjectMapperTests.cs

```diff
@@ -0,0 +1,99 @@
+﻿using AutoFixture;
+using Dfe.PrepareConversions.Data.Models.Establishment;
+using Dfe.PrepareConversions.Data.Models.Trust;
+using Dfe.PrepareConversions.Mappings;
+using FluentAssertions;
+using FluentAssertions.Execution;
+using System;
+using Xunit;
+
+namespace Dfe.PrepareConversions.Tests.Mappings;
+
+public class CreateSponsoredProjectMapperTests
+{
+   [Fact]
+   public void Should_Map_Establishment_With_PfiScheme_And_Trust_Into_Dto_CreateSponsoredProject()
+   {
+      // Arrange
+      var fixture = new AutoFixture.Fixture();
+
+      EstablishmentResponse establishment = new()
+      {
+         EstablishmentName = fixture.Create<string>(),
+         Urn = Guid.NewGuid().ToString(),
+         ViewAcademyConversion = new() { Pfi = "some-text" },
+         LocalAuthorityName = fixture.Create<string>(),
+         Gor = new EstablishmentResponse.Region() { Name = fixture.Create<string>() }
+      };
+
+      TrustDetail trust = new()
+      {
+         GiasData = new()
+         {
+            Ukprn = fixture.Create<string>(),
+            GroupId = fixture.Create<string>(),
+            GroupName = fixture.Create<string>(),
+            GroupType = fixture.Create<string>(),
+            CompaniesHouseNumber = fixture.Create<int>()
+         }
+      };
```

Comment:
This could be 
`TrustDetail trust = fixture.Build<TrustDetail>()
    .With(x => x.GiasData, fixture.Create<GiasData>())
    .Create();`
    
    or even just
    
    `TrustDetail trust = fixture.Create<TrustDetail>();`

Reply:
Fair point. Will change this next time around.